rudder-agent/debian/rules

```diff
@@ -83,7 +83,8 @@ binary-arch: install
 #	dh_installlogrotate
 #	dh_installmime
 #	dh_python
-	dh_installinit --no-start -R # restart after upgrade (instead of stop, upgrade, start)
+	dh_installinit --update-rcd-params=disable --no-start -R # restart after upgrade (instead of stop, upgrade, start)
+	dh_installinit --name=rudder --no-start -R # restart after upgrade (instead of stop, upgrade, start)
```

Comment:
Isn't the -R option a possible source of problem during upgrade ? From the doc : 

           Do not stop the init script until after the package upgrade has
           been completed. This is different than the default behavior, which
           stops the script in the prerm, and starts it again in the postinst.